config/webpack.prod.config.js

```diff
@@ -61,12 +61,19 @@ module.exports = merge(commonConfig, {
       // The babel-loader transforms newer ES2015+ syntax to older ES5 for older browsers.
       // Babel is configured with the .babelrc file at the root of the project.
       {
-        test: /\.(js|jsx)$/,
+        test: /\.(js|jsx|ts|tsx)$/,
         exclude: /node_modules\/(?!@edx)/,
         use: {
           loader: 'babel-loader',
           options: {
-            configFile: presets.babel.resolvedFilepath,
+            configFile: presets['babel-typescript'].resolvedFilepath,
+            // Caches result of loader to the filesystem. Future builds will attempt to read
+            // from the cache to avoid needing to run the expensive recompilation process
+            // on each run.
+            cacheDirectory: true,
+            plugins: [
+              require.resolve('react-refresh/babel'),
+            ],
```

Comment:
Are these relevant for the production build? 

The production build does not run on the Webpack Dev Server like webpack.dev.config.js does, so anything dealing with hot reloading (e.g., `react-refresh`, `cacheDirectory`) isn't really applicable to production builds, which is why they are not included in the existing webpack.prod.config.js file.

Reply:
Thanks for pointing that out, unneeded lines have been removed.